tests/view/crudeapi/test_comment.py

```diff
@@ -0,0 +1,105 @@
+import datetime
+import pytest
+import werkzeug
+from http import HTTPStatus
+from timeless.reservations.views import CommentView
+from timeless.reservations.views import Comment
+
+"""
+    Tests for CommentView
+    @todo #222:30min Finish CommentView implementation and CommentView tests. 
+     Methods for GET, POST and DELETE operation must be implemented, after the 
+     implementation remove the skip annotations from these tests.
+
+"""
+
```

Comment:
@paulodamaso You cannot test view the way you did it below, since `CommentView` inherited from `MethodView` and its methods `get`, `post`, `edit` and etc do not take arguments. See http://flask.pocoo.org/docs/1.0/views/#method-based-dispatching

So you have two options to test:
1) Take api client and do request with json
2) Use `test_request_context` which creates request with data you provided, see `http://flask.pocoo.org/docs/1.0/reqcontext/#manually-push-a-context`